src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -415,6 +413,14 @@ fn make_data(root: &Path, book: &Book, config: &Config, html_config: &HtmlConfig
                     json!("theme-tomorrow_night.js"));
     }
 
+    if html.search.enable {
+        if cfg!(feature = "search") {
+            data.insert("search_enabled".to_owned(), json!(true));
+        } else {
+            warn!("mdBook compiled without search support, ignoring `output.html.search.enable`");
```

Comment:
Good idea with adding a warning! Should we also say something like "please reinstall with `cargo install mdbook --force --features search` to use the search feature"?